engine/flink/management/dev-model/src/main/scala/pl/touk/nussknacker/engine/management/sample/LoggingService.scala

```diff
@@ -31,19 +36,39 @@ object LoggingService extends EagerService {
       override def invoke(context: Context)(
           implicit ec: ExecutionContext,
           collector: ServiceInvocationCollector,
-          componentUseCase: ComponentUseCase
+          componentUseContext: ComponentUseContext,
       ): Future[Any] = {
-        val msg = message.evaluate(context).renderedTemplate
-        level match {
-          case Level.TRACE => logger.trace(msg)
-          case Level.DEBUG => logger.debug(msg)
-          case Level.INFO  => logger.info(msg)
-          case Level.WARN  => logger.warn(msg)
-          case Level.ERROR => logger.error(msg)
+        if (isNotDisabled(componentUseContext)) {
+          val msg = message.evaluate(context).renderedTemplate
+          level match {
+            case Level.TRACE => logger.trace(msg)
+            case Level.DEBUG => logger.debug(msg)
+            case Level.INFO  => logger.info(msg)
+            case Level.WARN  => logger.warn(msg)
+            case Level.ERROR => logger.error(msg)
+          }
         }
         Future.successful(())
       }
 
+      private def isNotDisabled(componentUseContext: ComponentUseContext) =
+        componentUseContext
+          .deploymentData()
+          .flatMap(_.get("loggingComponentsDisabled"))
+          .forall(isDisabled => !isDisabled.toBoolean)
+
+      override def actionParametersDefinition: Map[ScenarioActionName, Map[ParameterName, ParameterConfig]] = Map(
+        ScenarioActionName.Deploy -> Map(
+          ParameterName("loggingComponentsDisabled") -> ParameterConfig(
```

Comment:
How about inverting logic? When I insert logging component to my scenario diagram I expect that logging is being done. Then, when I invoke "deploy" action, I can decide to supress logging.